NAMESPACE

```diff
@@ -55,6 +55,7 @@ export(mcmc_nuts_energy)
 export(mcmc_nuts_stepsize)
 export(mcmc_nuts_treedepth)
 export(mcmc_pairs)
+export(mcmc_parcoord)
```

Comment:
mcmc_parcoord_data function too?

Reply:
> Yes good point. I'll add that. In the case that the user specifies the nuts
parameters, do you think it makes sense to return a single data frame that
has a column indicating if there was a divergence, or a list of two data
frames already split?

> I guess that's also a general question for implementing the `_data`
functions. It would be nice to always return a single data frame for all of
those functions (like what you already did for ppc_intervals/ribbon) but I
wonder if we'll run into cases going forward when that's not the best
option. An alternative would be to always return a list of data frames and
sometimes (usually) that list will only have a single data frame. What do
you think? Whatever we choose it would be good to do it the same way for
all of them.

When possible, return just a dataframe. If a plot requires multiple data-frames, return them in a list. Definitely agree that we should try to be as consistent as possible.

Reply:
> Yes good point. I'll add that. In the case that the user specifies the nuts
parameters, do you think it makes sense to return a single data frame that
has a column indicating if there was a divergence, or a list of two data
frames already split?

With an indicator column.